cw-orch/src/osmosis_test_tube/core.rs

```diff
@@ -73,6 +74,12 @@ pub struct OsmosisTestTube<S: StateInterface = MockState> {
     pub app: Rc<RefCell<OsmosisTestApp>>,
 }
 
+/// Maps a RunnerError (from osmosis-test-tube), to a CwOrch Error
+/// As we don't have a custom Error type for this integration, we use this helper function to map errors instead
+pub fn map_err(e: RunnerError) -> CwOrchError {
```

Comment:
Shouldn't the error type exposed by `cw-orch` be in cw-orch itself instead of using the error type in cw-orch-core? That allows us to have a `#[from(...)]` variant on the error that's feature flagged and lets us remove this code.

Reply:
Yeah, let's go